assets/js/components/Board.js

```diff
@@ -5,26 +5,26 @@ import { GameRoundState } from '../actions'
 const Board = ({ gameState, gameId, board, updateRound, makeMove }) => {
     const mapSpaceSymbol = function(space) {
         switch(space){
-            case 1: return "X"
-            case 2: return "O"
+            case "PLAYER_1": return "X"
+            case "PLAYER_2": return "O"
         }
         return ""
     }
     const mapSpaceClass = function(space){
```

Comment:
I hadn’t commented before but I am a bit suprised to see CSS classes in places that not directly involved in rendering HTML/JSX. I feels like a presentation concern to me.

Reply:
I'm not sure what is the best practice in React, since everything is mixed there.
Should there be some separate js module that controls css style mappings?
Or should everything be split into tiny components which style themselves basing on props from outside?

Reply:
Sorry, I didn’t realise this is the actual component.